Xamarin.Forms.Controls/XamStore/StoreShell.xaml.cs

```diff
@@ -12,12 +12,13 @@ namespace Xamarin.Forms.Controls.XamStore
 	[XamlCompilation(XamlCompilationOptions.Compile)]
 	public partial class StoreShell : Shell
 	{
-		public StoreShell ()
-		{
-			InitializeComponent ();
+			public StoreShell ()
+			{
+				InitializeComponent ();
 
-			CurrentItem = _storeItem;
-		}
+				CurrentItem =  _storeItem;
+				Routing.RegisterRoute("demo", typeof(DemoShellPage));
```

Comment:
It looks like it's from PR https://github.com/xamarin/Xamarin.Forms/pull/5564.

Reply:
Yeah we need to review that one first please